docs/sphinx/source/modelchain.rst

```diff
@@ -433,13 +530,13 @@ function if you wanted to.
 
     def pvusa_ac_mc(mc):
```

Comment:
@kanderso-nrel I think it would be worth doing, as an example of how to use the multi-array functionality.

I'm not sure the `if mc.system.num_arrays == 1` condition is needed. PTAL.

Reply:
It is a little clunky but OK with me, especially as I am skeptical that many people use user-defined models anyway.  I guess it could use a comment or two explaining why the complexity is needed.

Here's an untested alternative that I'm not convinced is any cleaner:

```python
def pvusa_mc_wrapper(mc):
    if mc.system.num_arrays == 1:
        total_irrads = [mc.results.total_irrad]
    else:
        total_irrads = mc.results.total_irrad

    modeled_powers = tuple(
        pvusa(total_irrad['poa_global'], mc.results.weather['wind_speed'],
              mc.results.weather['temp_air'], array.module_parameters['a'],
              array.module_parameters['b'], array.module_parameters['c'],
              array.module_parameters['d'])
        for total_irrad, array
        in zip(total_irrads, mc.system.arrays)
    )
    if mc.system.num_arrays == 1:
        mc.results.dc = modeled_powers[0]
    else:
        mc.results.dc = modeled_powers

    return mc
```